libs/state/selections/src/lib/accumulation-observable.ts

```diff
@@ -45,7 +46,7 @@ export function createAccumulationObservable<T extends object>(
     ),
     tap(
       (newState) => (compositionObservable.state = newState),
-      (error) => console.error(error)
+      (error) => handleError(error)
     ),
     // @Notice We catch the error here as it get lost in between `publish` and `publishReplay`. We return empty to
     catchError((e) => EMPTY),
```

Comment:
example: 
`state. hold(actions,someAction$, () => api.getData())`
so, here, if api fails, we will not call api second time on trigger action